addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2217,6 +2268,20 @@ var SnippetsMenu = Widget.extend({
     },
 });
 
+function getClosestScrollableParent($target) {
+    console.log('TARGET Parent:', $target);
+    if ($target === null || $target.length === 0) {
+        return null;
+    }
+
+    console.log('FIND : ', $target[0].scrollHeight, $target[0].clientHeight);
+    if ($target[0].scrollHeight > $target[0].clientHeight) {
```

Comment:
Not related to be scrollable at all. Overflow hidden elements can satisfy this condition and they are the exact opposite of being scrollable... they specifically hide their overflowing content. Here the code is only about duplicating what's in tip.js: searching the element with overflow scroll/auto.